include/seqan3/alphabet/nucleotide/dna5.hpp

```diff
@@ -133,7 +139,7 @@ struct dna5
     }
     //!\}
 
-    //!\brief The size of the alphabet, i.e. the number of different values it can take.
+    //!\copydoc seqan3::dna4::value_size
     static constexpr rank_type value_size{5};
```

Comment:
> Where is dna4::value_size defined? I can't find it. Is this a default?

Well, it's defined in dna4.hpp... I am just copying the documentation from there.